python/generate/generate.sh

```diff
@@ -44,6 +44,9 @@ cp $ignore_file $sdk_output_folder
 sdk_version=$(cat $swagger_file | jq -r '.info.version')
 cat $config_file | jq -r --arg SDK_VERSION "$sdk_version" '.packageVersion |= $SDK_VERSION' > temp && mv temp $config_file
 
+# remove the verbose description
+cat $swagger_file | jq -r '.info.description |= "LUSID API"' > temp && mv temp $swagger_file
```

Comment:
Great solution, nice work! One thing I noticed when generating the SDK with this change is that we end up with 

```
LUSID API

LUSID API

The version of the OpenAPI document: 0.11.2291
Contact: info@finbourne.com
Generated by: https://openapi-generator.tech
```

Is there something else we can put instead of `LUSID API` to avoid duplication? I did notice that if you make it empty, it inputs `No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)  # noqa: E501` which is a shame.

Reply:
went with 
```
"""
    LUSID API

    FINBOURNE Technology  # noqa: E501

    The version of the OpenAPI document: 0.11.2285
    Contact: info@finbourne.com
    Generated by: https://openapi-generator.tech
"""
```